### DISCUSSION TITLE
New user confused by use of Environments :-(

### DISCUSSION THREAD

**Discussion by terryclancy** [Kong/insomnia#7978] [Help] [answered]
Dear Insomniacs,

I am new to Insomnia.rest and this is probably a newby question with an obvious answer – so sorry in advance ……..

I am using Insomnia 10.0.0:

 
![Insom1](https://github.com/user-attachments/assets/572529a8-ad4b-40da-89bf-364c6c995d1b)

The basic issue I have is that while I have been able to create new Environments, I am unable to “Activate Environment” or to “Manage Environments” because I can not find the following UI or similar (This screenshot is from https://www.youtube.com/watch?v=fzLPHpOP3Wc  which is certainly an older version of the product):

 ![Insom2](https://github.com/user-attachments/assets/74f6a100-1d2a-4f1e-9632-e63dde145176)

Instead I see  the following:

 
![Insom3](https://github.com/user-attachments/assets/85b8e232-008b-4cc2-9568-1881174001d7)

As a result the related environment variables are not appearing when I use Cntl “space” when defining my GET commands.

Please tell me how to “Activate Environment” and “Manage Environments”.

Thanks

Terry

**Comment by subnetmarco** [comment, marked as answer]
@terryclancy You can find the environment selector inside the collection:

![Screenshot 2024-09-22 at 2 38 20 PM](https://github.com/user-attachments/assets/d56e6c15-07ef-467f-b424-75e53e5c6d29)

The highlighted section in your screenshot is not the environment selector, but it is the the "Organization" selector that lists in plain text all the organizations that you can also see from the sidebar on the far left:

![Screenshot 2024-09-22 at 2 39 25 PM](https://github.com/user-attachments/assets/2bf4ac57-0d49-4ea2-8e71-d8499b870eea)

I hope this clarifies it!

**Comment by terryclancy** [comment]
Thank you subnetmarco for your assistance.   I now have it working and am slowly getting used to the UI :-)

Thanks again.
